src/javascript/_common/base/__tests__/client_base.js

```diff
@@ -1,3 +1,10 @@
+
+const Mock                         = require('mock-require');
+Mock('../../auth', {
+    isOAuth2Enabled: function() {
+        return false
+    }
+});
```

Comment:
This is to resolve test case issues where `@deriv-com/analytics` is using Growthbook module which uses MutationObserver, in which Mocha doesn't recognise and is unable to mock it
<img width="670" alt="Screenshot 2024-09-26 at 11 53 04 AM" src="https://github.com/user-attachments/assets/2e8ab5af-83d1-420c-89e1-2cbca62c76ec">